components/org.wso2.carbon.identity.api.server.tenant.management/org.wso2.carbon.identity.api.server.tenant.management.common/src/main/java/org/wso2/carbon/identity/api/server/tenant/management/common/TenantManagementConstants.java

```diff
@@ -43,6 +46,12 @@ public enum ErrorMessage {
         ERROR_CODE_TENANT_LIMIT_REACHED("TM-60019",
                 "Unable to create a tenant.",
                 "Maximum number of allowed tenants have been reached."),
+        ERROR_CODE_OWNER_NOT_FOUND("TM-60020",
+                "Unable to retrieve the tenant owner.",
+                "Tenant owner cannot be found for the provided id: %s."),
```

Comment:
which id ?

Reply:
Fixed in https://github.com/wso2/identity-api-server/pull/704/commits/cab5ddcccfde9ccfdfaf4a3f123ad205331f331f